Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -190,9 +191,11 @@ internal struct ChangeStreamTest: Decodable, FailPointConfigured {
                 case let .success(events):
                     var seenEvents: [BSONDocument] = []
                     for _ in 0..<events.count {
-                        let event = try changeStream.tryNext()?.get()
-                        expect(event).toNot(beNil(), description: self.description)
-                        seenEvents.append(event!)
+                        guard let event = try changeStream.nextWithTimeout() else {
```

Comment:
some of the newer tests were not receiving change stream events quickly enough on latest + sharded clusters, so I switched to the timeout method to ensure we actually get them.

conveniently, this fixes one occurrence of the `continueAfterFailure` not working on Linux problem you mentioned earlier @patrickfreed.